src/XMakeBuildEngine/BackEnd/BuildManager/BuildParameters.cs

```diff
@@ -974,31 +974,27 @@ private void EnsureToolsets()
 
         /// <summary>
         /// This method determines where MSBuild.Exe is and sets the NodeExePath to that by default.
+        /// Note - 4/14/16: This used to look at loaded assemblies and in the Framework folder for
+        /// MSBuild.exe. This would, however, find MSBuild.exe v4 that is included in the Framework
+        /// either from loaded assemblies (e.g. mscorlib) or the eventual fall-back to the framework
+        /// path. This was removed as it is certainly never correct.
```

Comment:
Not sure this note is worthwhile since removal of "certainly never correct" behavior seems un-noteworthy :)